docs/swarm/key-concepts.md

```diff
@@ -10,77 +10,79 @@ parent="engine_swarm"
-cluster are running in **Swarm mode**.
+The cluster management and orchestration features embedded in the Docker Engine
+are built using **SwarmKit**. Engines participating in a cluster are
+running in **Swarm mode**. You enable Swarm mode for the Engine by either
+initializing a Swarm or joining an existing Swarm.
 
-A **Swarm** is a cluster of Docker Engines where you deploy a set of application
-services. When you deploy an application to a Swarm, you specify the desired
-state of the services, such as which services to run and how many instances of
-those services. The Swarm takes care of all orchestration duties required to
-keep the services running in the desired state.
+A **Swarm** is a self-organizing cluster of Docker Engines where you deploy
+[services](#Services-and-tasks). The Docker Engine CLI includes the commands for
+Swarm management, such as adding and removing nodes. The CLI also includes the
+commands you need to deploy services to the Swarm and manage service
+orchestration.
 
-## Node
+When you run Docker Engine outside of Swarm mode, you execute container
+commands. When you run the Engine in Swarm mode, you orchestrate services.
 
-A **node** is an active instance of the Docker Engine in the Swarm.
+## Node
 
-When you deploy your application to a Swarm, **manager nodes** accept the
-service definition that describes the Swarm's desired state. Manager nodes also
-perform the orchestration and cluster management functions required to maintain
-the desired state of the Swarm. For example, when a manager node receives notice
-to deploy a web server, it dispatches the service tasks to worker nodes.
+A **node** is an instance of the Docker Engine participating in the Swarm.
 
-By default the Docker Engine starts one manager node for a Swarm, but as you
-scale you can add more managers to make the cluster more fault-tolerant. If you
-require high availability Swarm management, Docker recommends three or five
-Managers in your cluster.
+To deploy your application to a Swarm, you submit a service definition to a
+**manager node**. The manager node dispatches units of work called
+[tasks](#Services-and-tasks) to worker nodes.
 
-Because Swarm manager nodes share data using Raft, there must be an odd number
-of managers. The Swarm cluster can continue functioning in the face of up to
-`N/2` failures where `N` is the number of manager nodes.  More than five
-managers is likely to degrade cluster performance and is not recommended.
+Manager nodes also perform the orchestration and cluster management functions
+required to maintain the desired state of the Swarm. Manager nodes elect a single leader to conduct orchestration tasks.
 
 **Worker nodes** receive and execute tasks dispatched from manager nodes. By
 default manager nodes are also worker nodes, but you can configure managers to
-be manager-only nodes.
+be manager-only nodes. The agent notifies the manager node of the current
```

Comment:
> but you can configure managers to be manager-only nodes

@tonistiigi is that correct? My understanding was that you would need to use constraints to prevent tasks from running on managers, or set the manager nodes to "drain" to prevent them from receiving tasks.